worker/setup.py

```diff
@@ -172,16 +172,7 @@ def make_release_tree(self, base_dir, files):
 
 setup_args['tests_require'] = test_deps
 
-setup_args['extras_require'] = {
-    'test': [
-        'pep8',
-        # spellcheck introduced in version 1.4.0
-        'pylint>=1.4.0',
-        'pyenchant',
```

Comment:
We are loosing spell checking here. I can bring in https://crates.io/crates/typos if desired.

Reply:
I think this makes sense.